packages/flutter/lib/src/painting/text_painter.dart

```diff
@@ -33,14 +33,16 @@ class TextPainter {
   TextPainter({
     TextSpan text,
     TextAlign textAlign,
-    double textScaleFactor: 1.0
-  }) : _text = text, _textAlign = textAlign, _textScaleFactor = textScaleFactor {
+    double textScaleFactor: 1.0,
+    ui.TextOverflow textOverflow,
```

Comment:
ok well we definitely shouldn't have both if they're both going to be exposed in the framework API.